core/modules/comment/comment.module

```diff
@@ -1526,7 +1526,7 @@ function comment_config_create_validate(Config $staging_config, $all_changes) {
  */
 function comment_access($op, Comment $comment = NULL) {
   if ($op == 'edit') {
-    // Normalize edit op to update.
+    // Normalize 'edit' operation to 'update'.
     $op = 'update';
```

Comment:
...not sure why this did not kick in then. It should have kept things from breaking.